nixos/modules/services/web-apps/hedgedoc.nix

```diff
@@ -47,6 +47,12 @@ in
     package = lib.mkPackageOption pkgs "hedgedoc" { };
     enable = lib.mkEnableOption "the HedgeDoc Markdown Editor";
 
+    configureNginx = lib.mkOption {
```

Comment:
This option is useless, I guess you forgot adding `lib.mkIf cfg.configureNginx.enable { ... }` ?

Reply:
Ups... 😂